packages/api/src/controllers/access-control.ts

```diff
@@ -250,6 +271,30 @@ app.post(
   }
 );
 
+app.post("/suspend/:key", authorizer({ anyAdmin: true }), async (req, res) => {
+  const { key } = req.params;
+
+  const [keys] = await db.accessControlKey.find({
+    key,
+  });
+
+  if (!keys[0]) {
+    let doc = await db.accessControlKey.create({
+      id: uuid(),
+      key,
+      suspended: true,
+    });
+
+    res.status(200).json({ id: doc.id, suspended: true });
```

Comment:
do we need a return here or does `res.status` handle that

Reply:
It's already handled by .status().json()